autopilot-daemon/pkg/utils/functions.go

```diff
@@ -37,6 +38,20 @@ func GetClientsetInstance() *K8sClientset {
 	return k8sClientset
 }
 
+func OnNodeUpdate(labels map[string]string, key string) {
```

Comment:
I don't know why this is a "utils" function as it doesn't seem reusable.  Would it be better to put this code in the switch statement of WatchNodes?

Reply:
Right... yeah that's probably not the best place. I didn't want to "clutter" too much the main, so if you have some other suggestions I'll be happy to hear, otherwise I'll move that into main

Reply:
I think it'll be OK in `watchNode`. It's not a lot of code and it gives a nice illustation of what to do on an event